Drivers/Query/PdoQuery.php

```diff
@@ -147,11 +147,11 @@ protected function fetch($db, $query, array $args = array())
         foreach ($args as $arg => $val) {
             $stmt->bindValue($arg, $val, is_int($val) ? \PDO::PARAM_INT : \PDO::PARAM_STR);
         }
-
+        
         $stmt->execute();
-        $return = $stmt->fetchAll(\PDO::FETCH_ASSOC);
+        $return = $stmt->executeQuery();
```

Comment:
Hi @artdeoich 
Why do we have both lines ?

```php
$stmt->execute();
```
and

```php
$return = $stmt->executeQuery();
```

It seems that `execute()` is deprecated in favor of `executeQuery()`.
https://github.com/doctrine/dbal/blob/3.3.x/src/Statement.php#L152

So I think with your code, we are executing the query twice.